pkg/model/model.go

```diff
@@ -143,16 +143,22 @@ type Definition struct {
 	Caches  map[string]CacheDefinition `yaml:"caches"`
 }
 
-/***************
- * REFACTORING *
- ***************/
-
-// Processor process Dictionary and none, one or many element
+// Processor process Dictionary and produce none, one or many element
 type Processor interface {
 	Open() error
 	ProcessDictionary(Dictionary, Collector) error
 }
 
+// CoProcessor process many Dictionary from source and produce none, one or many element
+type CoProcessor interface {
```

Comment:
Maybe this is the new `Processor` interface and every Processor now needs to be a CoProcessor ?